cmf-cli/resources/template_feed/init/Builds/CD-Containers.yml

```diff
@@ -613,8 +612,10 @@ stages:
             script: |
               Remove-Item -Path $(DATABASE_NETWORK_SHARE)/Cmf.FullBackup.*.zip -Force -Verbose
               Remove-Item -Path $(DATABASE_NETWORK_SHARE)/*.bak -Force -Verbose
+              New-Item -ItemType Directory -Path "$(DATABASE_NETWORK_SHARE)" -Name "DailyBackup" -Force
               & tools\CmfDeploy.exe backup --parameters $env:AGENT_TEMPDIRECTORY/backup_parameters.json --packageSources .\DFPackages
               Move-Item -Path $(DATABASE_NETWORK_SHARE)/Cmf.FullBackup.*.zip -Destination $(DATABASE_NETWORK_SHARE)/DailyBackup  -Force
+              Move-Item -Path $(DATABASE_NETWORK_SHARE)/*.bak -Destination $(DATABASE_NETWORK_SHARE)/DailyBackup  -Force
```

Comment:
Why do you need the backup file? It's left there and cleared on the next run (l 614), though we can explicitly remove it now

Reply:
this simplifies the process because for the daily backup we only need the .bak file. if we had the zip, we would need to unzip first.